### PR TITLE
[Merged by Bors] - Support specifying a namespace to watch

### PR DESCRIPTION
**Pull request by maltesander**
## Description

try
```
cargo run -- run --watch-namespace test
```
or
```
export WATCH_NAMESPACE=test
cargo run -- run
```

fixes https://github.com/stackabletech/nifi-operator/issues/214

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/env_var_args.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
+
+=== PRODUCT_CONFIG
```
Comment (teozkr) [inline docs/modules/ROOT/pages/env_var_args.adoc:2]: Isn't this the same bunch of fields that we specify in `commandline_args.adoc`?

File: docs/modules/ROOT/pages/env_var_args.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+*Default value*: `/etc/stackable/nifi-operator/config-spec/properties.yaml`
+
+*Required*: false
+
+*Multiple values:* false
+
+[source]
+----
+export PRODUCT_CONFIG=/foo/bar/properties.yaml
+cargo run -- run
```
Comment (teozkr) [inline docs/modules/ROOT/pages/env_var_args.adoc:13]:
```suggestion
PRODUCT_CONFIG=/foo/bar/properties.yaml cargo run -- run
```

File: docs/modules/ROOT/pages/env_var_args.adoc (hunk 1 above)
Comment (maltesander) [reply docs/modules/ROOT/pages/env_var_args.adoc:2]: yeah

File: docs/modules/ROOT/pages/env_var_args.adoc (hunk 2 above)
Comment (maltesander) [reply docs/modules/ROOT/pages/env_var_args.adoc:13]: I have it like this in all other operators, so i would like to keep it like this.

File: docs/modules/ROOT/pages/env_var_args.adoc (hunk 1 above)
Comment (teozkr) [reply docs/modules/ROOT/pages/env_var_args.adoc:2]: I'd merge them into one file then.

Comment (maltesander) [reply docs/modules/ROOT/pages/env_var_args.adoc:2]: I mean they come together via clap, the new opa side container e.g. does not use clap and would only require the env variable? I think in that case it makes sense to separate it?

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**